doc/quick-start.rst

```diff
@@ -6,10 +6,148 @@ This document gives simple usage examples of Dune. You can also look at
 `examples <https://github.com/ocaml/dune/tree/master/example>`__ for complete
 examples of projects using Dune.
 
+Initializing projects
+=====================
+
+The following subsections illustrate basic usage of the ``dune init proj``
+subcommand. For more documentation see :ref:`initializing_components` and the
+inline help available from ``dune init --help``.
+
+.. _initializing-an-executable:
+
+Initializing an executable
+--------------------------
+
+To initialize a project to build an executable program, run the following
```

Comment:
"To initialize a project that will build an executable program".  ?? 
Just to remove the repetitive "to" ... but it's fine either way, really.

Reply:
Ah that's better. Thanks :)